gitpod-ws.code-workspace

```diff
@@ -1,77 +1,398 @@
 {
-    "folders": [
```

Comment:
Would be lovely with a comment here stating that this bit is automatically generated - and include how to run the script to update it.